conanfile.py

```diff
@@ -75,8 +85,46 @@ class Xrpl(ConanFile):
         'zlib/1.2.11',
     ]
 
+    def set_version(self):
+        BUILDINFO = f"src/ripple/protocol/impl/BuildInfo.cpp"
```

Comment:
Change this string to `f"{self.recipe_folder}/src/ripple/protocol/impl/BuildInfo.cpp"` so that dependencies can be installed from an out-of-source build directory.